src/test/java/org/folio/logintest/CredentialExistenceTest.java

```diff
@@ -65,12 +69,17 @@ public static void setup(final TestContext context) {
     vertx.deployVerticle(RestVerticle.class.getName(), restVerticleDeploymentOptions, res -> {
       try {
         TenantAttributes ta = new TenantAttributes().withModuleTo("mod-login-1.1.0");
-        tenantClient.postTenant(ta, handler -> saveCredential()
-          .onComplete(v -> {
-            if (v.succeeded()) {
-              async.complete();
-            }
-          }));
+        tenantClient.postTenant(ta, handler -> {
+          Future<Void> f1 = saveCredential(EXISTING_CREDENTIALS_USER_ID, "password");
+          Future<Void> f2 = saveCredential(EXISTING_EMPTY_CREDENTIALS_USER_ID, "");
+
+          CompositeFuture.all(f1, f2)
+            .onComplete(v -> {
+              if (v.succeeded()) {
+                async.complete();
+              }
+            });
```

Comment:
Please also report a failure to immediately fail the test with good exception reporting. The easiest way is
```suggestion
          CompositeFuture.all(f1, f2).onComplete(context.asyncAssertSuccess(done -> async.complete()));
```